plugins/modules/system/launchd.py

```diff
@@ -0,0 +1,482 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Martin Migasiewicz <migasiew.nk@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: launchd
+author:
+- Martin Migasiewicz (@martinm82)
+short_description:  Manage macOS services
+description:
+- Manage launchd services on target macOS hosts.
+options:
+    name:
+      description:
+      - Name of the service.
+      type: str
+      required: true
+    state:
+      description:
+      - C(started)/C(stopped) are idempotent actions that will not run
+        commands unless necessary.
+      - Launchd does not support C(restarted) nor C(reloaded) natively.
+        These will trigger a stop/start (restarted) or an unload/load
+        (reloaded).
+      - C(restarted) unloads and loads the service before start to ensure
+        that the latest job definition (plist) is used.
+      - C(reloaded) unloads and loads the service to ensure that the latest
+        job definition (plist) is used. Whether a service is started or
+        stopped depends on the content of the definition file.
+      type: str
+      choices: [ reloaded, restarted, started, stopped, unloaded ]
+    enabled:
+      description:
+      - Whether the service should start on boot.
+      - B(At least one of state and enabled are required.)
+      type: bool
+    force_stop:
+      description:
+      - Whether the service should not be restarted automatically by launchd.
+      - Services might have the 'KeepAlive' attribute set to true in a launchd configuration.
+        In case this is set to true, stopping a service will cause that launchd starts the service again.
+      - Set this option to C(yes) to let this module change the 'KeepAlive' attribute to false.
+      type: bool
+      default: no
+notes:
+- One option other than name is required.
+requirements:
+- A system managed by launchd
+- The plistlib python library
+'''
+
+EXAMPLES = r'''
+- name: Make sure spotify webhelper is started
+  community.general.launchd:
+    name: com.spotify.webhelper
+    state: started
+  become: yes
```

Comment:
```suggestion
```
I think this is unnecessary and is about how to use ansible in general. You could add a note (to the note section) that users need proper privileges to work with launchd, etc. and remove this directive from the examples